internal/serve/httphandler/tss_handler.go

```diff
@@ -5,67 +5,110 @@ import (
 
 	"github.com/stellar/go/support/log"
 	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/go/txnbuild"
 	"github.com/stellar/wallet-backend/internal/apptracker"
 	"github.com/stellar/wallet-backend/internal/serve/httperror"
 	"github.com/stellar/wallet-backend/internal/tss"
 	"github.com/stellar/wallet-backend/internal/tss/router"
+	tssservices "github.com/stellar/wallet-backend/internal/tss/services"
 	"github.com/stellar/wallet-backend/internal/tss/store"
 	"github.com/stellar/wallet-backend/internal/tss/utils"
 )
 
 type TSSHandler struct {
-	Router            router.Router
-	Store             store.Store
-	AppTracker        apptracker.AppTracker
-	NetworkPassphrase string
+	Router             router.Router
+	Store              store.Store
+	AppTracker         apptracker.AppTracker
+	NetworkPassphrase  string
+	TransactionService tssservices.TransactionService
 }
 
 type Transaction struct {
 	Operations []string `json:"operations" validate:"required"`
+	TimeOut    int64    `json:"timeout" validate:"required"`
```

Comment:
nit: can we rename this to Timebounds since that is what we use in our docs and other places

Reply:
good point! made this change